backend/api/views/tests/test_statistics_r.py

```diff
@@ -210,10 +168,13 @@ def test_get_report_records_statistics(client):
 
 
 def test_get_total(client):
-    id_list = [
-        create_factory(client)
-        for _ in range(10)
-    ]
+    # note: asterisk sign a.k.a. * for tuple unpacking is not supported in list comprehension
+    # the feature was taken out of PEP 448 for readability
+    # see Variations session of PEP 448 https://peps.python.org/pep-0448/#variations
```

Comment:
PEP 448 is interesting. @EagleC0318 @yanghaochang104 @Neilxx you may want to read it a bit.